fedn/fedn/clients/reducer/control.py

```diff
@@ -74,12 +74,18 @@ def get_latest_model(self):
     def get_model_info(self):
         return self.statestore.get_model_info()
 
+    def get_client_info(self):
+        return self.statestore.list_clients()
+
     def get_events(self):
         return self.statestore.get_events()
 
     def drop_models(self):
         self.statestore.drop_models()
 
+    def update_client_data(self, client_data, status, role):
```

Comment:
I think this should belong in network.py

Reply:
done!